pdfs/src/Pdf/FromLatex.hs

```diff
@@ -0,0 +1,206 @@
+
+module Pdf.FromLatex where
+
+import qualified Data.ByteString as BS
+import Path.IO
+import Path
+import System.Directory
+import System.IO.Temp
+import System.Process.Typed
+
+escapeInLatex :: Text -> Text
+escapeInLatex text = text
+
+type Locale = Text
+
+data Date = Date
+    { year  :: Int
+    , month :: Int
+    , day   :: Int
+    }
+
+dateToString :: Locale -> Date -> Text
+dateToString _ (Date y m d) = show d <> "." <> show m <> "." <> show y
+
+data StudentInfo = StudentInfo
+    { siName        :: Text
+    , siDateOfBirth :: Date
+    , siSections    :: [Section]
+    }
+
+data Section = Section
+    { sEducator :: Text
+    , sDiploma  :: Diploma
+    , sCourses  :: [Course]
+    }
+
+data Diploma = Diploma
+    { dEducatorUrl     :: Text
+    , dEducationPeriod :: (Int, Int)
+    , dDiplomaId       :: Text
+    , dDegreeLevel     :: Text
+    , dMajor           :: Text
+    , dMinor           :: Text
+    , dPartTime        :: Bool
+    }
```

Comment:
AFAIS it should also contain 3 different dates:
https://www.notion.so/serokell/Application-for-creating-verifiable-certificates-5a7802e4a07d408396008ddfdb324883